README.md

```diff
@@ -40,4 +41,4 @@ Hopefully by now, you have your own spreadsheet of different companies that you
 
 ## Other Resources
 * [LSH Resources List](https://github.com/Ladies-Storm-Hackathons/Resources): Ladies Storm Hackathons is a diversity organization for women in tech.  This repository contains resources on interviewing, negotiating, attending hackthons and diversity focused student organizations.
-* [Hackathon Hackers](https://github.com/HackathonHackers): Comprehensive list of different HH subgroups as well as a list of other amazing opportunities
\ No newline at end of file
+* [Hackathon Hackers](https://github.com/HackathonHackers): Comprehensive list of different HH subgroups as well as a list of other amazing opportunities
```

Comment:
Hmm I can't seem to distinguish whats the difference between what was already there and what you added.  Could you either:
- Leave a comment explaining what change was made.
- If nothing was changed, then can you please revert back this change.